cmd/sensors/src/lib.rs

```diff
@@ -681,12 +677,18 @@ fn sensors(context: &mut ExecutionContext) -> Result<()> {
 
     let mut reader: Box<dyn SensorReader> = match subargs.backend {
         Some(Backend::Hiffy) => {
+            if core.is_dump() {
+                bail!("cannot use hiffy backend on dump");
+            }
```

Comment:
nit: perhaps the CLI help text should also mention this?

Reply:
Sure, done in feea70d58273a78d0d167b05ba2c69a0b5a17244